src/commands/dailyCheckIn/scheduleCheckIn.js

```diff
@@ -0,0 +1,191 @@
+
+const { PermissionFlagsBits, ApplicationCommandOptionType, Client, CommandInteraction, ApplicationCommandOptionWithChoicesMixin } = require("discord.js");
+const dayjs = require('dayjs');
+const utc = require('dayjs/plugin/utc');
+const weekday = require('dayjs/plugin/weekday');
+const localizedFormat = require('dayjs/plugin/localizedFormat');
+dayjs.extend(utc);
+dayjs.extend(weekday);
+dayjs.extend(localizedFormat);
+
+
+const fakeScheduleEntry = {
+}
+
+module.exports = {
+    /**
+     * Parse and save schedule date to the database
+     * @param {Client} client 
+     * @param {CommandInteraction} interaction 
+     */
+    callback: async (client, interaction) => {
+
+        const userId = interaction?.user?.id;
+        const userTag = interaction?.user?.tag;
+
+        await interaction.deferReply({ ephemeral: true });
+
+        try {
+
+            // command should include a user
+            if (userId === undefined || userTag === undefined) {
+                await interaction.editReply({
+                    ephemeral: true,
+                    content: 'Could not process command'
+                });
+                return;
+            }
+
+
+            const rawDays = interaction.options.get('days')?.value;
+            const rawTime = interaction.options.get('time')?.value;
+            let parsedDays = [];
+            const validDays = ['sunday', 'monday', 'tuesday', 'wednesday', 'thursday', 'friday', 'saturday'];
+            const abbreviations = {
+                'm': 'monday',
+                't': 'tuesday',
+                'w': 'wednesday',
+                'th': 'thursday',
+                'f': 'friday',
+                's': 'saturday',
+                'su': 'sunday',
+            }
```

Comment:
- I suggest having some sort of documentation (either in the description of the command or in the read me) that says what abbreviations are valid. "H" could be considered valid for Thursdays, but the bot rejects this. 

- For consistency and lack of confusion, make Saturday's abbreviation `Sa`.

Reply:
updated the description of the command